r/r.code-workspace

```diff
@@ -1,17 +0,0 @@
-{
-	"folders": [
-		{
-			"path": "."
-		}
-	],
-	"settings": {
-		// R configurations
-		"[r]": {
-			"editor.formatOnPaste": true,
-			"editor.formatOnSave": true
-		},
-		"r.sessionWatcher": true,
-		"r.bracketedPaste": true,
-		"r.rterm.linux": "/usr/local/bin/radian",
-	}
-}
```

Comment:
Could we leave this here in case engineers want to keep using vscode?

Reply:
Will do. I'll add it back in with any other changes as it seems that the docker image doesn't use the previous build image from the PR for the cache (I suspect it is using develop image or somesuch).